ci/build.sh

```diff
@@ -3,8 +3,17 @@ set -eu -o pipefail
 
 export VIRTUAL_ENV_DIR=venv
 
+VERSION=$(grep "version =" setup.cfg | sed 's/^.*= //')
```

Comment:
Can save yourself a `grep` with a denser `sed` invocation:

```suggestion
VERSION=$( sed -n 's/^\s*version\s*=\s*\(.*\)/\1/p' setup.cfg )
```

Reply:
I'm going to leave the grep/sed combo, since we're using that most everywhere, and the consistency is worth more than doing it all in a single tool.